wcfsetup/install/files/lib/system/database/table/column/AbstractDecimalDatabaseTableColumn.class.php

```diff
@@ -23,7 +23,7 @@ abstract class AbstractDecimalDatabaseTableColumn extends AbstractDatabaseTableC
     /**
      * @inheritDoc
      */
-    public function getDefaultValue()
+    public function getDefaultValue(): mixed
```

Comment:
This should likely use a more specific value by leveraging the covariance of the return types. I believe `string|null` (please not `?string` for clarity) would be appropriate.